src/components/createPostModal/index.js

```diff
@@ -31,8 +31,8 @@ const CreatePostModal = ({triggerUpdate, setTriggerUpdate}) => {
     return (
         <>
             <section className="create-post-user-details">
-                <div className="profile-icon"><p>AJ</p></div>
-                <div className="post-user-name"><p>Alex J</p></div>
+                <div className="profile-icon"><p>{currentUser.firstName?.[0]}{currentUser.lastName?.[0]}</p></div>
```

Comment:
Can we move the getting of the initials outside of the JSX, and have it in a variable. It's a little confusing just seeing this inline.